### ISSUE TITLE
WCAG 1.3.3 ✅ (Supports): Sensory Characteristics

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#5105]
## WCAG 1.3.3: Sensory Characteristics

**Level:** A
**Principle:** Perceivable
**Result:**  ✅ Supports

While OLH only partially supports it, any users affected can choose to use accessibility mode which does support it, therefore overall, Janeway supports this criterion.

- Material - ✅  Supports
- Clean - ✅ Supports
- A11y Mode - ✅  Supports
- OLH - 🟠  Partially Supports

### Description
This issue tracks the audit and implementation of WCAG 2.2 Success Criterion 1.3.3: Sensory Characteristics.

### Resources
- [WCAG 2.2 Success Criterion 1.3.3](https://www.w3.org/WAI/WCAG22/Understanding/sensory-characteristics.html)
>Goal
Instructions are understandable by more people.
What to do
Describe controls by name, not just by appearance or location.
Why it's important
People who are blind or have low vision need non-visual instructions.

### Audit Checklist
- [x] Review previous Audit Result
- [x] Review work identified in previous audit and list open issues.
- [x] Check for any other relevant open issues.
- [x] Retest open issues, are these issues still present?
- [x] Test for any new issues that may have been introduced.
- [x] Determine compliance status, add this to issue name.
- [x] Add any new issues to the Accessibility project for triage.
- [x] Replace previous audit in milestone with this update.

### Previous audit
> ❌ Sensory Characteristics
We do not rely on sound, size or orientation to convey information. We are in the process of upgrading our Captcha to make it more accessible. This audit found that we have used icons and typographical marks to convey information without providing non-visual alternatives. Colour is being used in the Material theme to show invalid user input.

#4126

 ### Progress since last audit

- openlibhums/janeway#3945
- openlibhums/janeway#4210
- openlibhums/janeway#4230
- openlibhums/janeway#4058
- openlibhums/janeway#4215
- openlibhums/janeway#4200
- openlibhums/janeway#4227
- openlibhums/janeway#3936
- openlibhums/janeway#4268 

 ### Audit Notes

Janeway does not to use sound, size or orientation to convey information. Janeway does not use sounds. Orientation is only handled in as much as the page reflows to match the screen-size with both portrait and landscape use anticipated.  The appearance of elements is handled through CSS and may be different for instance through custom styling, so cannot be used within the platform to describe elements.

The position of an element is based on template structure, so could have been used in some places to reference that element.  This audit included a search for position words.  None were found to be used in this way. (Technique G96).

Graphical symbols could have been used to convey information without text alternatives.  The icons have been update across the codebase since the last audit. This audit included a codebase search for all icons to confirm none had been missed.

Captcha has been updated to a version which includes accessibility bypasses. Icons have been updated across the codebase to either mark as decorative or provide semantic alternatives. 

### Testing
- I ran a search of the codebase for location words (right, left, top, bottom, up, down, below, above). These are not in use to describe locations anywhere in the front of house.
- I ran an icon search (for font-awesome icons)
  - ✅ Material
  - ✅ Clean
  - 🟠 : OLH - one icon on the article page did not have text equivalent.   This was a very minor issue so I've put through a fix now and will pause this audit issue in the hope that it can be merged and I can update to "supports".
  - A11y mode: ✅ 
  - Common templates: ✅ 
  - javascript ✅ 
 - additional searches for other icons
   - `&times;` which is primarily used as a `x` on close buttons.   All pass except the OLH asset `toastr.js` which has a close button with no other text. This is a framework asset, used 'as is'. 

## User Content

We are aware that some user-content includes Greek letters and/or Roman numerals which require markup in order to be correctly interpreted by screen readers.  This content is outside our control, however we are looking at ways to assist our users to ensure it is accessible:
- openlibhums/janeway#4223
- openlibhums/janeway#4238
- openlibhums/janeway#4243

 ### Work arising

Work is required to make OLH compliant. 
- The icon on the article page needs updating (PR awaiting approval).
- We need to look into whether toastr.js is still in use on the front of house for OLH, and either update or remove it.
